<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta http-equiv="Content-Style-Type" content="text/css">
  <title>Publications -- Rhodes</title>
  <meta name="Generator" content="Cocoa HTML Writer">
  <meta name="CocoaVersion" content="2113.3">
  <style type="text/css">
    p.p2 {margin: 0.0px 0.0px 12.0px 0.0px; font: 12.0px Times; color: #000000; -webkit-text-stroke: #000000}
    span.s1 {font-kerning: none}
    span.s2 {font: 12.0px Times; text-decoration: underline ; font-kerning: none; color: #0000e9; -webkit-text-stroke: 0px #0000e9}
    span.s3 {font: 12.0px Times; text-decoration: underline ; font-kerning: none; color: #0000e9}
  </style>
</head>
<body>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; color: #000000; -webkit-text-stroke: #000000"><span class="s1"><b>Recent Publications (Phylogenetics, Statistical Models, related topics)</b></span></h2>

<p class="p2"><span class="s1">"Parameter identifiability of a multitype pure-birth model of speciation," with D. Dragomir and E.S. Allman, submitted, 2022. 
<a href="https://arxiv.org/abs/??????"><span class="s2">ArXiv</span></a> <span class="Apple-converted-space"> </span></span></p>

<p class="p2"><span class="s1">"The Tree of Blobs of a Species Network: Identifiability under the Coalescent,'' with E.S. Allman, H. Banos, and J.D. Mitchell, submitted 2022 
<a href="https://arxiv.org/abs/2205.03466"><span class="s2">ArXiv</span></a> <span class="Apple-converted-space"> </span></span></p>

<p class="p2"><span class="s1">"Testing Multispecies Coalescent Simulators using Summary Statistics,'' with E.S. Allman and H. Banos, to appear, IEEE/ACM Transactions in Computational Biology and Bioinformatics <a href="https://arxiv.org/abs/1908.01424"><span class="s2">ArXiv</span></a><span class="Apple-converted-space"> </span></span></p>

<p class="p2"><span class="s1">"Identifiability of species network topologies from genomic sequences using the logDet distance," with E.S. Allman and H. Banos, Journal of Mathematical Biology, 84:35,2022. 
<a href="https://rdcu.be/cKUs5"><span class="s2">JMB</span></a><span class="Apple-converted-space"> </span></span>,
<a href="https://arxiv.org/abs/2108.01765"><span class="s2">ArXiv</span></a> <span class="Apple-converted-space"> </span></span></p> 
<p class="p2"><span class="s1">"Parameter identifiability for a profile mixture model of protein evolution,'' with S. Yourdkhani and E.S. Allman, Journal of Computational Biology, 28 (6), 2021. <a href="https://arxiv.org/abs/2007.02185"><span class="s2">ArXiv</span></a><span class="Apple-converted-space"></span></span></p>
<p class="p2"><span class="s1">"Gene tree discord, simplex plots, and statistical tests under the coalescent,'' with E.S. Allman and J.D. Mitchell, Systematic Biology, 71 (4) 2022, 929-942. <a href="https://academic.oup.com/sysbio/article/71/4/929/6131696?guestAccessKey=bc92695e-061a-484e-af1b-b450b8d608ca"><span class="s2">PDF</span></a><span class="Apple-converted-space"> </span></span></p>
<p class="p2"><span class="s1">"MSCquartets 1.0: Quartet methods for species trees and networks under the multispecies coalescent model in R," with E.S. Allman, H. Banos, and J.D. Mitchell, Bioinformatics, 10 (2020). <a href="https://doi.org/10.1093/bioinformatics/btaa868"><span class="s2">PDF</span></a><span class="Apple-converted-space"> </span></span></p>
<p class="p2"><span class="s1">"Inferring metric trees from weighted quartets via an intertaxon distance,'' with S. Yourdkhani, Bulletin of Mathematical Biology, 82 (7), 2020, 97. <a href="https://arxiv.org/abs/2002.04564"><span class="s2">PDF</span></a><span class="Apple-converted-space"> </span></span></p>
<p class="p2"><span class="s1">"NJst and ASTRID are not statistically consistent under a random model of missing data,'' with M.G. Nute and T. Warnow. <a href="https://arxiv.org/abs/2001.07844"><span class="s2">PDF</span></a><span class="Apple-converted-space"> </span></span></p>

<p class="p2"><span class="s1">"NANUQ: A method for inferring species networks from gene trees under the coalescent model,'' with E.S. Allman and H. Banos, Algorithms in Molecular Biology, 14(24) 2019. <a href="https://almob.biomedcentral.com/articles/10.1186/s13015-019-0159-2"><span class="s2">PDF</span></a><span class="Apple-converted-space"> </span></span></p>
<p class="p2"><span class="s1">"Topological metrizations of trees, and new quartet methods of tree inference,'' IEEE/ACM Transactions in Computational Biology and Bioinformatics, 17(6), 2020, 2107-2118. <a href="https://arxiv.org/abs/1704.02004"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">"Hypothesis testing near singularities and bondaries," with J D. Mitchell and E.S. Allman, Electronic Journal of Statistics 13(1), 2019, 1250-1293. <a href="https://projecteuclid.org/euclid.ejs/1561687407"><span class="s2">PDF</span></a><span class="Apple-converted-space"> </span></span></p>
<p class="p2"><span class="s1">"Maximum likelihood estimation of the Latent Class Model through model boundary decomposition," with E. S. Allman, H. Banos, R. Evans, S. Hosten, K. Kubjas, D. Lemke, and P. Zwiernik, Journal of Algebraic Statistics, 10 (2019) 3-18. <a href="https://jarhodesuaf.github.io/papers/MLboundaries.pdf"><span class="s2">PDF</span></a><span class="Apple-converted-space"> </span></span></p>
<p class="p2"><span class="s1">"Species tree inference from genomic sequences using the log-det distance," with E.S. Allman and C. Long, SIAM Journal on Applied Algebra and Geometry 3 (2019) 107-127. <a href="https://jarhodesUAF.github.io/papers/LogDetCoal.pdf"><span class="s2">PDF</span></a><span class="Apple-converted-space"> </span></span></p>
<p class="p2"><span class="s1">“Split Probabilities and Species Tree Inference Under the Multispecies Coalescent Model,” with E.S. Allman and J.H. Degnan, Bulletin of Mathematical Biology, 80 (2018), 64-103. <a href="https://jarhodesUAF.github.io/papers/SplitProbs.pdf"><span class="s3">PDF</span></a></span></p>
<p class="p2"><span class="s1">“Species tree inference from gene splits by Unrooted STAR methods,” with E.S. Allman and J.H. Degnan, IEEE/ACM Transactions on Computational Biology and Bioinformatics, 15, no. 1, (2018) 337-342. <a href="https://jarhodesUAF.github.io/papers/USTAR_consist_TCBB.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">“Split scores: a tool to quantify phylogenetic signal in genome-scale data,” with E.S. Allman and L.S. Kubatko, Systematic Biology, 66 (2017), 620-636. <a href="https://arxiv.org/abs/1608.00942"><span class="s2">PDF</span></a><span class="Apple-converted-space"> </span></span></p>
<p class="p2"><span class="s1">“Statistically-Consistent k-mer Methods for Phylogenetic Tree Reconstruction,” with E.S. Allman and S. Sullivant, Journal of Computational Biology, 24, no. 2 (2017) 153-171. <a href="https://jarhodesUAF.github.io/papers/kmer_paper_color.pdf"><span class="s2">PDF</span></a> and<br>
<a href="http://www.dms.uaf.edu/~eallman/Papers/kmerSupp.html"><span class="s2">Supplementary Materials</span></a><span class="Apple-converted-space"> </span></span></p>
<p class="p2"><span class="s1">“Phylogenetic trees and Euclidean embeddings,” with M. Layer,<span class="Apple-converted-space">  </span>Journal of Mathematical Biology, 74, (2017) 99-111. <a href="https://jarhodesUAF.github.io/papers/EuclideanEmbeddings.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">“There are no caterpillars in a wicked forest,” with J.H. Degnan, Theoretical Population Biology, 105 (2015) 17-23. <a href="https://jarhodesUAF.github.io/papers/NoCats.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">“Parameter identifiability of discrete Bayesian networks with hidden variables,” with E.S. Allman, E. Stanghellini, and M. Valtorta, Journal of Causal Inference, 3, no.2, (2015) 189-205. <a href="https://jarhodesUAF.github.io/papers/paramDBM.pdf"><span class="s2">PDF</span></a><span class="Apple-converted-space"> </span></span></p>
<p class="p2"><span class="s1">“Tensors of Nonnegative Rank Two,'' with E.S. Allman, B. Sturmfels, and P. Zwienik, Linear Algebra and its Application, 473 (2015) 37-53. <a href="https://jarhodesUAF.github.io/papers/NNrank2.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">“A Semialgebraic Description of the General Markov Model on Phylogentic Trees,'' with E.S. Allman and A. Taylor, SIAM Journal on Discrete Mathematics, 28, no. 2 (2014) 736-755. <a href="https://jarhodesUAF.github.io/papers/SemialgGM.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">“Tensor Rank, Invariants, Inequalities, and Applications,'' with E.S. Allman, P. Jarvis, and J. Sumner, SIAM Journal on Matrix Analysis and Applications, 34, no. 3 (2013) 1014-1045. <a href="https://jarhodesUAF.github.io/papers/SIMAXtensor.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">“Species tree inference by the STAR method, and generalizations,'' with E.S. Allman and J.H. Degnan, Journal of Computational Biology, 20, no. 1 (2013), 50-61. <a href="https://jarhodesUAF.github.io/papers/STARandGeneralizations.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">“When do phylogenetic mixture models mimic other phylogenetic models?,'' with E.S. Allman and S. Sullivant, Systematic Biology, 61, no. 6 (2012), 1049-1059. <a href="https://jarhodesUAF.github.io/papers/MixturesMimic.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">“Determining species tree topologies from clade probabilities under the coalescent,” with E. S. Allman and J. H. Degnan, Journal of Theoretical Biology, 289 (2011), 96-106. <a href="https://jarhodesUAF.github.io/papers/YJTBI6577Clades.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">“Identifiability of Large Phylogenetic Mixture Models,” with S. Sullivant, Bulletin of Mathematical Biology, 74 (2012), 212-231. <a href="https://jarhodesUAF.github.io/papers/LargeMixtures_BMB2012.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">“Parameter identifiability in a class of random graph mixture models,” with E. S. Allman and C. Matias, Journal of Statistical Planning and Inference, 141 (2011), 1719-1736. <a href="https://jarhodesUAF.github.io/papers/JSPI4479RandomGraphs.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">“Identifying the Rooted Species Tree from the Distribution of Unrooted Gene Trees under the Coalescent,” with E. S. Allman and J. H. Degnan, Journal of Mathematical Biology, 62 no. 6 (2011), 833-862. <a href="https://jarhodesUAF.github.io/papers/STfromUnrootedGTs.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">"Identifiability of 2-tree mixtures for group-based models,'' with E. S. Allman, S. Petrovic, and S. Sullivant, IEEE/ACM Transactions on Computational Biology and Bioinformatics, 8 no. 3 (2011), 710-722. <a href="https://jarhodesUAF.github.io/papers/TwoTrees.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">"Trees, Fast and Accurate," with E. S. Allman, Science, 327 no. 5971 (2010), 1334-1335. <a href="https://jarhodesUAF.github.io/papers/TreesFast_Science.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">"Estimating Trees from Filtered Data: Identifiability of Models for Morphological Phylogenetics,'' with E. S. Allman and M. T. Holder, Journal of Theoretical Biology 263 (2010), 108-119. <a href="https://jarhodesUAF.github.io/papers/MkConsist.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">"A concise proof of Kruskal's theorem on tensor decomposition,'' Linear Algebra and Its Applications 432 (2010), 1818-1824. <a href="https://jarhodesUAF.github.io/papers/kruskal_decomp.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">"Identifiability of parameters in latent structure models with many observed variables,'' with E. S. Allman and C. Matias, Annals of Statistics, 37 no. 6A (2009), 3099--3132. <a href="https://jarhodesUAF.github.io/papers/Latent.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">"The identifiability of covarion models in phylogenetics," with E. S. Allman, IEEE/ACM Transactions in Computational Biology and Bioinformatics, 6 no. 1 (2009), 76--88. <a href="https://jarhodesUAF.github.io/papers/CovId.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">"Identifiability of a Markovian model of molecular evolution with Gamma-distributed rates,'' with E. S. Allman and C. Ane, Advances in Applied Probability,<b> 40</b> no. 1 (2008), 229--249. <a href="https://jarhodesUAF.github.io/papers/GamId.pdf"><span class="s2">PDF</span></a><br>
</span></p>
<p class="p2"><span class="s1">"Identifying evolutionary trees and substitution parameters for the general Markov model with invariable sites,'' with E. S. Allman, Mathematical Biosciences, <b>211</b> no.1 (2008), 18--33. <a href="https://jarhodesUAF.github.io/papers/GMI.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">"Molecular phylogenetics from an algebraic viewpoint,'' with E. S. Allman, Statistica Sinica, <b>17</b> no. 4 (2007), 1299--1316. <a href="https://jarhodesUAF.github.io/papers/StatSinica.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">"Phylogenetic ideals and varieties for the general Markov model,'' with E. S. Allman, Advances in Applied Mathematics, <b>20</b> no. 2 (2008), 127--148. <a href="https://jarhodesUAF.github.io/papers/PhyloIdealsVarsforGM.pdf"><span class="s2">PDF</span></a><br>
</span></p>
<p class="p2"><span class="s1">"Phylogenetic invariants,'' with E. S. Allman, in <i>Reconstructing Evolution: New Mathematical and Computational Advances</i>, ed. by O. Gascuel and M. Steel. Oxford University Press, 2007, 108--147. <a href="https://jarhodesUAF.github.io/papers/Gascuel-chap04.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">"Phylogenetics,'' with E. S. Allman, in <i>Modeling and Simulation of Biological Networks</i>, ed. by R. Laubenbacher. Proceedings of Symposia in Applied Mathematics, American Mathematical Society, 2007. <a href="https://jarhodesUAF.github.io/papers/AMSsymposium.pdf"><span class="s2">PDF</span></a><br>
</span></p>
<p class="p2"><span class="s1">"The identifiability of tree topology for phylogenetic models, including covarion and mixture models,'' with E. S. Allman, Journal of Computational Biology, <b>13</b>, (2006), 1101--1113. <a href="https://jarhodesUAF.github.io/papers/Identifiability.pdf"><span class="s2">PDF</span></a><br>
</span></p>
<p class="p2"><span class="s1">"Phylogenetic invariants for stationary base composition,'' with E. S. Allman, Journal of Symbolic Computation, <b>41</b> (2006), 138--150. <a href="https://jarhodesUAF.github.io/papers/PhyloInvforStationary.pdf"><span class="s2">PDF</span></a><br>
</span></p>
<p class="p2"><span class="s1">"Quartets and parameter recovery for the general Markov model of sequence mutation,'' with E. S. Allman, Applied Mathematics Research Express, <b>4</b> (2004), 107--131. <a href="https://jarhodesUAF.github.io/papers/Quartets.pdf"><span class="s2">PDF</span></a></span></p>
<p class="p2"><span class="s1">"Phylogenetic invariants for the general Markov model of sequence mutation,'' with E. S. Allman, Mathematical Biosciences, <b>186</b> (2003), 113--144. <a href="https://jarhodesUAF.github.io/papers/PhyloInvforGM.pdf"><span class="s2">PDF</span></a><br>
</span></p>
<p class="p2"><span class="s1"> </span></p>
</body>
</html>
